common/src/main/java/com/wynntils/functions/InventoryFunctions.java

```diff
@@ -238,10 +248,21 @@ protected List<String> getAliases() {
         }
     }
 
+    public static class HeldItemShinyStatFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            ItemStack itemStack = InventoryUtils.getItemInHand();
+            Optional<ShinyStat> shinyStatOpt = Models.Shiny.getShinyStat(itemStack);
+            if (shinyStatOpt.isEmpty()) return "";
+
+            return shinyStatOpt.get().name() + ": " + shinyStatOpt.get().value();
+        }
+    }
```

Comment:
I would rather see this separated into a name and value function. The all function can stay.

Reply:
Should we introduce a new generic type, like capped? A "holder" type, or whatever, that has a string as a description, and a number. And then HeldItemShinyStat can return such a holder, and we can get name and value from that holder. I think it is a general construct enough that it can be useful in other situations as well.

I am slightly worried about the exploding matrix if we start making special functions for too much detail, like if I split this into two, then someone will want to have a "get helmet" version and then it too needs to versions, etc. Better to try and raise the abstraction level of our function language.

Reply:
Okay, I guess that can work.